Sitecore/jss
# Proper building of query string inside EditingRenderMiddleware

## Description / Motivation
I want to override default `resolvePageUrl` and add GET parameter.
I use the way described in the official documentation:
https://doc.sitecore.com/xp/en/developers/hd/200/sitecore-headless-development/override-default-protocols-using-the-editingrendermiddleware-in-next-js-jss-apps.html?utm_medium=rss&utm_source=dev.sitecore.net

```
const handler = new EditingRenderMiddleware({
  resolvePageUrl: (serverUrl: string, itemPath: string) => `${serverUrl}${itemPath}?someMyKey=${myKeyValue}&`,
}).getHandler();
```

Why do I need this parameter: I need to indicate to some components that the request came from EE/Horizon. 
When I add only `someMyKey=${myKeyValue}` it doesn't work because of adding `?timestamp=${Date.now()}`. The result string will have 2 `?` signs, which is not right. I can do some tricks and add `?someMyKey=${myKeyValue}&`(with ampersand an at the end). It will give me 2 parameters: `someMyKey` and `?timestamp`. 

As `timestamp` parameter is added only to avoid cache, it doesn't matter how it will be named: `timestamp` or `?timestamp`. It makes this change very minor. But if we can have a proper GET parameter name, why don't have it?


## Testing Details

1. `yarn install`
1. `yarn build`
1. Copy only `editing-render-middleware.js`
1. Add logging of URL (to make sure that it was changed in the proper way)
1. Try different actions in Experience Editor (adding and removing different components)
1. Review logs: look for "timestamp"

## Types of changes
